Routing/Geometry.swift

```diff
@@ -37,3 +37,29 @@ public class BoundingBox: NSObject {
         super.init()
     }
 }
+
+extension Point {
+    public override func isEqual(object: AnyObject?) -> Bool {
+        guard let otherObject = object as? Point else {
+            return false
+        }
+        return self == otherObject
+    }
+}
+
+extension BoundingBox {
+    public override func isEqual(object: AnyObject?) -> Bool {
+        guard let otherObject = object as? BoundingBox else {
+            return false
+        }
+        return self == otherObject
+    }
+}
+
+func ==(lhs: Point, rhs: Point) -> Bool {
+    return lhs.x == rhs.x && lhs.y == rhs.y
```

Comment:
For anyone else wondering...
![screenshot 2016-03-02 11 15 10](https://cloud.githubusercontent.com/assets/117291/13458921/1ca39642-e068-11e5-90f9-49ae70bf855e.png)